examples/agentworkflow/weather.ts

```diff
@@ -0,0 +1,32 @@
+import { AgentWorkflow } from "@llamaindex/core/agent";
```

Comment:
if it doesn't add new deps, we can just move it to workflow package
```suggestion
import { AgentWorkflow } from "@llamaindex/workflow";
```